checklists/afd_checklist.en.json

```diff
@@ -164,6 +164,7 @@
             "guid": "28b9ee82-b2c7-45aa-bc98-6de6f59a095d",
             "id": "A01.27",
             "severity": "High",
+            "graph" : "resources | where type == 'microsoft.network/frontdoorwebapplicationfirewallpolicies' | project policyName=name, policyId=id,policySku=sku.name, links=properties.securityPolicyLinks, enabledState=properties.policySettings.enabledState, mode=properties.policySettings.mode | mvexpand links | extend securityPolicy=links.id | extend securityPolicyParts=split(securityPolicy, '/') | extend profileId=strcat_array(array_slice(securityPolicyParts, 0, -3), '/') | project id=profileId, compliant=((enabledState=='Enabled') and (mode=='Prevention')), enabledState, mode",
```

Comment:
Hey @asukumar22 thanks again! Would it be possible to change this and use the `=~` operator for comparisons? Thanks!

Reply:
Hi Jose! Thanks for the comments! :)  updated it with case insensitive operator.